autoinstrumentation/utils/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# This utils image is intended to be used as a base image for OpenTelemetry Operator
+# autoinstrumentation images. The utils will allow autoinstrumentation packages to be
+# copied (via a go based cp command) from the init container to the final destination volume.
+FROM golang as utils
```

Comment:
This Dockerfile still results in an image that is distributed with the `golang` base image right? What is `golang` built on top of? I think this would need to be a multistage build to avoid this where the last stage is `FROM scratch`.

Reply:
You are correct. I refactored the code and it now lives in autoinstrumentation/cp/Dockerfile. 
I would really like us to follow a similar pattern shown in this file whether or not the code is based off of rust or golang. This way downstream auto-instrumentation vendors don't have to worry about the implementation because it is abstracted enough for them not to care about it.
Let me know your thoughts if you have them.